Tests/TestingTests/PlanTests.swift

```diff
@@ -421,6 +421,15 @@ struct PlanTests {
     #expect(plan.stepGraph.subgraph(at: typeInfo.fullyQualifiedNameComponents + CollectionOfOne("reserved1(reserved2:)")) != nil)
   }
 
+  @Test("Containing suite types without @Suite are synthesized")
+  func containingSuiteSynthesis() async {
+    let plan = await Runner.Plan(selecting: OutermostSuite.self)
+    let testNames = plan.steps.map(\.test.name)
+    #expect(testNames.contains("OutermostSuite"))
```

Comment:
Is `PlanTests` part of this list also?

Reply:
Yes. I nested the fixture suite within `PlanTests` right before posting the PR and forgot to come back and add it here for clarity. It isn't strictly necessary to validate that the fix works (checking for the presence of `OutermostSuite` is sufficient). But including `PlanTests` would be better, so I'll add that and also restructure the `#expect` so that it requires a strict order for the test names.